src/SFML/Graphics/RenderTexture.cpp

```diff
@@ -61,6 +61,9 @@ bool RenderTexture::create(unsigned int width, unsigned int height, bool depthBu
     // We disable smoothing by default for render textures
     setSmooth(false);
 
+    // Mark the texture as being owned by a RenderTexture
+    m_texture.m_ownedByRenderTexture = true;
```

Comment:
Since this bug is only specific to the FBO implementation, maybe make the fix also specific to the FBO implementation. There is no dependency when texture copying is used in the default implementation, so this fix isn't necessary there. Might consider renaming the variable to something like `m_fboAttachment` in that case.